qds_sdk/cloud/azure_cloud.py

```diff
@@ -0,0 +1,208 @@
+    '''
+
+    def __init__(self):
+        self.compute_config = {}
+        self.location = {}
+        self.network_config = {}
+        self.storage_config = {}
+
+    def set_cloud_config(self,
+                         compute_client_id=None,
+                         compute_client_secret=None,
+                         compute_subscription_id=None,
+                         compute_tenant_id=None,
+                         use_account_compute_creds=None,
+                         location=None,
+                         storage_access_key=None,
+                         storage_account_name=None,
+                         disk_storage_account_name=None,
+                         disk_storage_account_resource_group_name=None,
+                         persistent_security_groups=None,
+                         bastion_node_public_dns=None,
+                         vnet_name=None,
+                         subnet_name=None,
+                         vnet_resource_group_name=None,
+                         master_elastic_ip=None):
+        '''
+
+        Args:
+            compute_client_id: Client id for azure cluster
+
+            compute_client_secret: Client secret key for azure cluster
+
+            compute_subscription_id: Subscription id for azure cluster
+
+            compute_tenant_id: Tenant id for azure cluster
+
+            use_account_compute_creds: Set it to true to use the account’s compute
+                credentials for all clusters of the account.The default value is false
+
+            location: Location for azure cluster
+
+            storage_access_key: Storage access key for azure cluster
+
+            storage_account_name: Storage account name for azure cluster
+
+            disk_storage_account_name: Disk storage account name for azure cluster
+
+            disk_storage_account_resource_group_name: Disk storage account resource group
+                namefor azure cluster
+
+            persistent_security_groups: security group to associate with each node of the cluster.
+                Typically used to provide access to external hosts
+
+            bastion_node_public_dns: public dns name of the bastion node.
+                Required only if cluster is in private subnet of a EC2-VPC
+
+            vnet_name: vnet name for azure
+
+            subnet_name: subnet name for azure
+
+            vnet_resource_group_name: vnet resource group name for azure
+
+            master_elastic_ip: It is the Elastic IP address for attaching to the cluster master
+
+        '''
+
+
+        self.set_compute_config(use_account_compute_creds, compute_tenant_id, compute_subscription_id, compute_client_id, compute_client_secret)
+        self.set_location(location)
+        self.set_network_config(bastion_node_public_dns, persistent_security_groups, master_elastic_ip, vnet_name, subnet_name, vnet_resource_group_name)
```

Comment:
nit: Split the fn arguments in multiple lines (not necessarily one line for one argument).